src/Sylius/Bundle/UiBundle/composer.json

```diff
@@ -40,6 +40,7 @@
         "symfony/security-core": "^5.4.21 || ^6.0",
         "symfony/security-bundle": "^5.4.21 || ^6.0",
         "symfony/templating": "^5.4.21 || ^6.0",
+        "symfony/ux-twig-component": "^2.9.1",
```

Comment:
Maybe handle it as an optional requirement?

Reply:
As `UiBundle` is kind of our internal Bundle it won't change too much, as we'll require this package in the `AdminBundle` anyway (and in the ShopBundle too, I guess) 🤔. I don't see a use-case for `UiBundle` in any other project than Sylius :D.

Reply:
It's used by Monofony. Yeah I know that we need to create a better package, but for now it's still used on Monofony. And the Grid documentation mention the UI bundle sometimes => https://github.com/Sylius/SyliusGridBundle/blob/1.13/docs/your_first_grid.md

Reply:
And I'd like to split this UiBundle into small component => sylius/ui-translations, sylius/ui-twig-extension (with twig functions liek these ones https://github.com/Sylius/Sylius/tree/1.13/src/Sylius/Bundle/UiBundle/Twig)

Reply:
We agreed to postpone making Twig Component as an optional dep :).